docs/community-operators-troubleshooting.md

```diff
@@ -0,0 +1,219 @@
+# Troubleshooting the Community Operator Pipeline
```

Comment:
@haripate This document looks like a copy-paste from the troubleshooting guide for partners. Since community support is not the same as ISV please revisit this doc and remove any ISV-related stuff (things like "contact Red Hat Support." should be replaced by rasing an issue in the repository or something similar.)

Also, make sure all tasks are covered here. I compared a number of tasks in this doc with a number of tasks in actuall pipeline and number are different. Please review a list and add missing tasks.